src/htable.cpp

```diff
@@ -924,24 +925,24 @@ HeadedTable::HeadedTable(QWidget *parent, int opts) : QWidget(parent)
     // synchronize horizontal scrolling of head and body
     connect(body->horizontalScrollBar(), SIGNAL(valueChanged(int)), head,
             SLOT(scrollSideways(int)));
-    fontChange(font()); // *** need for init
+    fontChange(); // *** need for init
                         //	vp=new VPointer((QWidget *)QObject::parent());
                         //	vp->hide();		//memory leak!!
 }
 
 HeadedTable::~HeadedTable() {}
 
 // ok : move to Qttableview
-void HeadedTable::fontChange(const QFont &oldFont)
+void HeadedTable::fontChange()
 {
     // DEBUG("fontChange()\n");
-    int fontHeight = fontMetrics().height() + 1;
+    int fontHeight = QApplication::fontMetrics().height() + 1;
```

Comment:
The widget gets its font from QApplication but it's safer to work with the widget font because we might change it independently from QApplication at some point -- although we don't do so now. Anyhow, there no need to this change, IMO.